pkg/config/setup/config.go

```diff
@@ -1714,6 +1714,7 @@ func LoadDatadogCustom(config pkgconfigmodel.Config, origin string, secretResolv
 func LoadCustom(config pkgconfigmodel.Config, origin string, secretResolver optional.Option[secrets.Component], additionalKnownEnvVars []string) (*pkgconfigmodel.Warnings, error) {
 	warnings := pkgconfigmodel.Warnings{}
 
+	log.Info("Starting to load the configuration")
```

Comment:
I think it would also be useful to have a log message directly before `LoadProxyFromEnv` as well as directly before `ResolveSecrets` (and only if the secretResolver optional is available, so within the `if` condition). There are the key places where we don't have a lot of visibility on what's happening at agent start-up, and having logs around this would help debugging issues when flares get sent.

Reply:
I added one in `LoadProxyFromEnv` ; for `ResolveSecrets` there's already one inside the function itself